api/app/src/test/kotlin/packit/integration/controllers/PacketControllerTest.kt

```diff
@@ -1,14 +1,56 @@
 package packit.integration.controllers
 
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
 import packit.integration.IntegrationTest
+import packit.model.Packet
+import packit.repository.PacketRepository
+import java.time.Instant
+import kotlin.test.assertEquals
 
 class PacketControllerTest : IntegrationTest()
 {
+    @Autowired
+    lateinit var packetRepository: PacketRepository
+
+    val packet = Packet(
+        "1", "test", "test name",
+        mapOf("name" to "value"), false, Instant.now().epochSecond
+    )
+
+    @BeforeEach
+    fun `init`()
+    {
+        packetRepository.save(packet)
+    }
+
+    @AfterEach
+    fun teardown()
+    {
+        packetRepository.delete(packet)
```

Comment:
to avoid having to do this manual teardown each time, I think go ahead and add
```
@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
@Transactional
```

to the base `IntegrationTest` class.